src/style.css

```diff
@@ -1 +1,54 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of styling `.stars__star`, which is likely an element tag, use a class to apply these styles. This will make your CSS more maintainable and avoid potential conflicts if you have other `div` elements that shouldn't have these styles.